modules/apps/batch-planner/batch-planner-test/src/testFunctional/tests/SupportExportObjectEntriesAtInstancelevel.testcase

```diff
@@ -0,0 +1,329 @@
+				expectedValue = "/o/headless-batch-engine/v1.0/import-task/com.liferay.object.rest.dto.v1_0.ObjectEntry",
+				fileName = "export.batch-engine-data.json",
+				jsonObject = "$.actions.createBatch.href");
+
+			BatchEngine.assertExportedFileContainsCorrectObject(
+				expectedValue = "com.liferay.object.rest.dto.v1_0.ObjectEntry",
+				fileName = "export.batch-engine-data.json",
+				jsonObject = "$.configuration.className");
+		}
+	}
+
+	@priority = 4
+	test CanExportSystemObjectItems {
+		property portal.acceptance = "true";
+
+		task ("And Given 'Account (v1_0 - Liferay Headless Admin User)' as Entity Type in Data Migration Center --> Export File") {
+			ImportExport.openImportExportAdmin();
+
+			ImportExport.gotoExport();
+
+			ImportExport.selectEntity(entityType = "Account (v1_0 - Liferay Headless Admin User)");
+		}
+
+		task ("When exporting file in JSONT format with all fields selected") {
+			ImportExport.configureExport(
+				entityType = "Account (v1_0 - Liferay Headless Admin User)",
+				exportFields = "Attribute Code",
+				exportFileFormat = "JSONT");
+
+			Button.click(button = "Export");
+
+			WaitForElementPresent(locator1 = "ImportExport#EXECUTION_SUCCESS");
+		}
+
+		task ("Then all items fields values, actions node, configuration node available in the dowloaded file") {
+			ImportExport.unzipDownloadedExportFile();
+
+			var response = AccountAPI.getAccounts();
+
+			var dateCreated = JSONPathUtil.getProperty(
+				property = "$.items..dateCreated",
+				response = ${response});
+			var dateModified = JSONPathUtil.getProperty(
+				property = "$.items..dateModified",
+				response = ${response});
+			var externalReferenceCode = JSONPathUtil.getProperty(
+				property = "$.items..externalReferenceCode",
+				response = ${response});
+			var accountEntryId = JSONPathUtil.getProperty(
+				property = "$.items..id",
+				response = ${response});
+			var logoURL = JSONPathUtil.getProperty(
+				property = "$.items..logoURL",
+				response = ${response});
+			var name = JSONPathUtil.getProperty(
+				property = "$.items..name",
+				response = ${response});
+			var type = JSONPathUtil.getProperty(
+				property = "$.items..type",
+				response = ${response});
+			var fieldsValuesArray1 = "${dateCreated};dateCreated,${dateModified};dateModified,${externalReferenceCode};externalReferenceCode";
+			var fieldsValuesArray2 = "${accountEntryId};id,${logoURL};logoURL,${name};name,${type};type";
+
+			var totalFieldsValuesArray = "${fieldsValuesArray1},${fieldsValuesArray2}";
+
+			for (var i : list ${totalFieldsValuesArray}) {
+				var expectedValue = StringUtil.extractFirst(${i}, ";");
+				var fieldName = StringUtil.extractLast(${i}, ";");
+
+				BatchEngine.assertExportedFileContainsCorrectObject(
+					expectedValue = ${expectedValue},
+					fileName = "export.batch-engine-data.json",
+					jsonObject = "$.items..${fieldName}");
+			}
+
+			BatchEngine.assertExportedFileContainsCorrectObject(
```

Comment:
Same for actions node

Reply:
I modified the code to get .actions..method to assert all actions nodes in the export file